.npmrc

```diff
@@ -1 +1,2 @@
-registry=https://registry.npmjs.com/
\ No newline at end of file
+registry=https://registry.npmjs.com/
```

Comment:
Please revert your change to this file.  I don't know whether this is causing the build problem or not, but there should be no need to change this file.

Reply:
done